File: javascript/2161.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var a = parseInt(input)
var raiz;

if(a === 0){
    raiz = 3.0;
    console.log(raiz.toFixed(10))
}else {
    raiz = 6.0;
    
    for(let i = 0; i < a - 1; i++){
        raiz = 6 + 1 / raiz;
    }
    raiz = 3 + 1 / raiz;
    console.log(raiz.toFixed(10))
}

File: python/2161.py
n = int(input())

if n == 0:
    raiz = 3.0
    print('{:.10f}'.format(raiz))
else:
    raiz = 6.0
    for num in range(n - 1):
        raiz = 6 + 1 / raiz
    raiz = 3 + 1 / raiz
    print('{:.10f}'.format(raiz))